### PR TITLE
Fix some pyflakes warnings

### PR DESCRIPTION
**Pull request by mwichmann**
These are of the kind that would not be fixed by a code reformat (i.e. Black).  These should all be trivial. They're nearly all in unit tests (all but two), since I haven't cleaned up as many there in the past.

Signed-off-by: Mats Wichmann <mats@linux.com>

## Contributor Checklist:

* [X] I have created a new test or updated the unit tests to cover the new/changed functionality.
* [ ] I have updated `master/src/CHANGES.txt` directory (and read the `README.txt` in that directory)
* [ ] I have updated the appropriate documentation

### CODE REVIEW

File: src/engine/SCons/TaskmasterTests.py [hunk 1]
```diff
@@ -935,7 +935,7 @@ def raise_UserError():
         except SCons.Errors.UserError:
             pass
         else:
-            raise TestFailed("did not catch expected UserError")
+            raise AssertionError("did not catch expected UserError")
```
Comment (bdbaddog) [inline src/engine/SCons/TaskmasterTests.py]: ? why change assertion type?

Comment (mwichmann) [reply src/engine/SCons/TaskmasterTests.py]: TestFailed is undefined.  This was a guess at something to replace it.

Comment (mwichmann) [reply src/engine/SCons/TaskmasterTests.py]: ... one assumes the else clause has never been touched, since we don't see this as an error.

Comment (bdbaddog) [reply src/engine/SCons/TaskmasterTests.py]: Looks like TestFailed() is a deprecated part of unittest.   I think the sugguested replacement would be ```self.fail()```

Comment (mwichmann) [reply src/engine/SCons/TaskmasterTests.py]: ok, can change that if you like.

Comment (bdbaddog) [reply src/engine/SCons/TaskmasterTests.py]: Please do. Thanks!

**Pull request merged**